src/lib/Behat/StudioUtility.php

```diff
@@ -0,0 +1,286 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUi\Behat;
+
+use Behat\Mink\Element\NodeElement;
+use Exception;
+use PHPUnit\Framework\Assert;
+
+/**
+ * @deprecated These methods are used by StudioUI bundle, but should not be used in AdminUI
+ */
+trait StudioUtility
```

Comment:
These methods are used (some heavily) in Studio, but I don't want to include them in UtilityContext. Some of them could be refactored, some extracted to Elements (Dropdown, Button etc.). Refactoring of Studio will be a good opportunity to get rid of them.